client/src/main/java/hudson/plugins/swarm/Options.java

```diff
@@ -32,6 +32,9 @@
                     "in which case the missing portion will be auto-configured like the default behavior")
     public String tunnel;
 
+    @Option(name = "-webSocket", usage = "Connect using the WebSocket protocol")
```

Comment:
The `-tunnel` option and the `-webSocket` option are [mutually exclusive in Remoting](https://github.com/jenkinsci/remoting/blob/142a5ef53ef80df3bf7b0f10ca9e555238ec33e3/src/main/java/hudson/remoting/jnlp/Main.java#L83-L87), so I think we should disallow them at the Swarm client layer as well.

```suggestion
    @Option(name = "-webSocket", usage = "Connect using the WebSocket protocol", forbids="-tunnel")
```